### PR TITLE
[WFLY-19256][WFLY-19259] Move CDI and EL to final releases in WildFly Preview

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.redhat.com/browse/WFLY-19256
https://issues.redhat.com/browse/WFLY-19259

This builds on blocker PR #17825 because the upgrade needs to be reflected in the documentation that PR adds.

____

<--- THIS SECTION IS AUTOMATICALLY GENERATED BY WILDFLY GITHUB BOT. ANY MANUAL CHANGES WILL BE LOST. --->

> Wildfly issue links:
> * [WFLY-18932](https://issues.redhat.com/browse/WFLY-18932)

<--- END OF WILDFLY GITHUB BOT REPORT --->

More information about the [wildfly-bot[bot]](https://github.com/wildfly/wildfly-github-bot)

### CODE REVIEW

**Comment by bstansberry** [comment]
I put the Critical label on this under the assumption there would be no CI issues. If there aren't there's value in having WFP include the final spec versions, so we should prioritize review/merge.

**Review by jamezp — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @jamezp

File: docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc [hunk 1]
```diff
@@ -62,17 +62,100 @@ This feature pack is the WildFly Preview analogue to main WildFly's ``wildfly@ma
...
+|Jakarta Mail| 2.1 |10 & 11
+
+|Jakarta Messaging| 3.1 |10 & 11
+
+| Jakarta MVC
+(_preview stability only_)| 2.1| N/A xref:note2[^2^]
+
+|Jakarta Persistence| 3.2.0-M2 |11
+
+|Jakarta RESTful Web Services| 3.1 |10
```
Comment (scottmarlow) [inline docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc:127]: Consider changing to `4.0` https://jakarta.ee/specifications/restful-ws/4.0/ when possible.

File: docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc [hunk 2]
```diff
@@ -62,17 +62,100 @@ This feature pack is the WildFly Preview analogue to main WildFly's ``wildfly@ma
...
+
+|Jakarta Security| 4.0.0-M2 |11
+
+|Jakarta Server Pages| 3.1 |10
```
Comment (scottmarlow) [inline docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc:131]: [Jakarta Server Pages](https://jakarta.ee/specifications/pages/) should now be Jakarta Pages.

Comment (scottmarlow) [reply docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc:131]: 4.0 is under development as well.

File: docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc [hunk 3]
```diff
@@ -62,17 +62,100 @@ This feature pack is the WildFly Preview analogue to main WildFly's ``wildfly@ma
...
+
+|Jakarta Servlet| 6.1.0-M2 |11
+
+|Jakarta SOAP with Attachments| 1.3 |10 xref:note1[^1^]
```
Comment (scottmarlow) [inline docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc:135]: Should this be 3.0?  https://jakarta.ee/specifications/soap-attachments/3.0/

Comment (bstansberry) [reply docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc:135]: @scottmarlow Yes. I suspect that is wrong elsewhere too. I'll fix this.

File: docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc (hunk 2 above)
Comment (bstansberry) [reply docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc:131]: @scottmarlow Thanks; I'll fix the name, which is probably wrong elsewhere.

**Comment by bstansberry** [comment]
@scottmarlow Thanks for the feedback. Regarding Jakarta Authentication and Jakarta RESTful Webservices, this document shows what is integrated in WildFly Preview 32, not what's being worked on in Jakarta. So hopefully the WF 33 version of this will include updates to those two. Almost certainly it will for REST as we'd have moved in WF 32 if they'd have published an RC while the ballot was in progress.

**Comment by bstansberry** [comment]
@scottmarlow 

PR to fix this in WF main: https://github.com/wildfly/wildfly/pull/17850
And you already approved the PR to fix this in the WF 32 release announcement. :)

Thanks!

**Pull request merged**